commons/utils/support_bundle_utils.py

```diff
@@ -213,18 +213,25 @@ def collect_support_bundle_k8s(local_dir_path: str, scripts_path: str = cm_const
     flg = False
     resp = m_node_obj.execute_cmd(cmd=cm_cmd.CLSTR_LOGS_CMD.format(scripts_path), read_lines=True)
     for line in resp:
-        if ".tar" in line:
+        if "date" in line:
```

Comment:
Can we check SB_PREFIX as well ?

Reply:
> Can we check SB_PREFIX as well ?

Not needed I believe. As this  line will just have date info only. And that's what the intention is to fetch